core/state/trie_prefetcher_extra_test.go

```diff
@@ -123,11 +123,8 @@ func BenchmarkPrefetcherDatabase(b *testing.B) {
 			commit(levelDB, snaps, db)
 			b.Logf("Root: %v, kvs: %d, block: %d (committed)", root, count, block)
 		}
-		if previous != root {
-			require.NoError(db.TrieDB().Dereference(previous))
-		} else {
-			b.Fatal("root did not change")
-		}
+		require.Equal(root, previous, "root and previous should be equal")
```

Comment:
they should not be equal

Reply:
Lol for some reason both of them pass. Changed to NotEqual.